api/URLSearchParams.json

```
@@ -133,32 +184,25 @@
                 "version_added": "53"
               },
               "firefox_android": {
-                "version_added": "53"
+                "version_added": "54"
               },
               "ie": {
                 "version_added": false
               },
-              "nodejs": [
-                {
-                  "version_added": "7.10.0"
-                },
-                {
-                  "version_added": "7.5.0",
-                  "partial_implementation": true,
-                  "notes": "Only string as <code>init</code> object supported."
-                }
-              ],
+              "nodejs": {
+                "version_added": "7.10.0"
+              },
               "opera": {
                 "version_added": "48"
               },
               "opera_android": {
-                "version_added": "45"
+                "version_added": "48"
```

I think this ought to be 45 still (Opera desktop and Android version numbers aren't lined up, unfortunately).

```suggestion
                "version_added": "45"
```

---

Good catch! 
I didn't know that they started drifting apart at some point.